homeassistant/components/utility_meter/sensor.py

```diff
@@ -183,6 +184,13 @@ async def async_setup_platform(
     for conf in discovery_info.values():
         meter = conf[CONF_METER]
         conf_meter_source = hass.data[DATA_UTILITY][meter][CONF_SOURCE_SENSOR]
+        conf_meter_unique_id = hass.data[DATA_UTILITY][meter].get(CONF_UNIQUE_ID)
+        conf_sensor_tariff = conf.get(CONF_TARIFF)
```

Comment:
Maybe do like this to make the unique_id more readable:
```suggestion
        conf_sensor_tariff = conf.get(CONF_TARIFF, "single_tariff")
```